### PR TITLE
opers.c: fix typos

### PR DESCRIPTION
**Pull request by ChristopherRussell**
More instances of the typo that I corrected in #1411. Thanks to @fingolfin for pointing them out.

Edit: I had a look for some more typos involving "an"

### CODE REVIEW

File: hpcgap/extern/gc/extra/AmigaOS.c [hunk 1]
```diff
@@ -153,7 +153,7 @@ ptr_t GC_get_stack_base()
         } /* for */
 #       ifdef __GNUC__
            if (!found_segment) {
-             ABORT("Can`t find correct Segments.\nSolution: Use an newer version of ixemul.library");
```
Comment (fingolfin) [inline hpcgap/extern/gc/extra/AmigaOS.c]: Code inside `extern` comes from 3rd parties, and we should not modify it.

Comment (ChristopherRussell) [inline hpcgap/extern/gc/extra/AmigaOS.c]: Okay, I have reverted this.

File: lib/pcgs.gd [hunk 2]
```diff
@@ -278,7 +278,7 @@ DeclareAttribute( "PcSeries", IsPcgs );
 ##  <Prop Name="IsPcgsElementaryAbelianSeries" Arg='pcgs'/>
 ##
 ##  <Description>
-##  returns <K>true</K> if the pcgs <A>pcgs</A> refines an elementary abelian
+##  returns <K>true</K> if the pcgs <A>pcgs</A> refines a elementary abelian
```
Comment (fingolfin) [inline lib/pcgs.gd]: Are you sure about that one?

My "feel of the language" (admittedly of a non-native speaker) makes me think "an elementary" is actually right, so does the rule on it I learned on it (when pronounced, the word starts with a sounding vowel).
 "Google fight" seems to agree: 15 million hits for the former and tons of books with "an elementary introduction" in the title; only 400k hits for "a elementary", and many of them seem to be for "big A elementary".

Or perhaps this is an American vs. British english thing?

**Comment by fingolfin** [comment]
Other than my remark on "a(n) elementary", I am happy with this PR being merged. And overall appreciate that you took the time to make it, thank you!

**Comment by james-d-mitchell** [comment]
"An elemantary" is correct in both US and UK English

**Comment by ChristopherRussell** [comment]
I agree and I have reverted to an elementary. Whilst doing this, I found and changed an instance of 'a elementary' in lib/grp.gi

**Pull request merged**